src/renderer.c

```diff
@@ -329,7 +354,7 @@ static inline GlyphMetric *font_get_glyph(RenFont *font, unsigned int glyph_id,
   return font->glyphs.metrics[subpixel_idx][row] ? &font->glyphs.metrics[subpixel_idx][row][col] : NULL;
 }
 
-static RenFont *font_group_get_glyph(RenFont **fonts, unsigned int codepoint, int subpixel_idx, SDL_Surface **surface, GlyphMetric **metric) {
+static RenFont *font_group_get_glyph(RenFont **fonts, unsigned int codepoint, int subpixel_idx, bool load_bitmap, SDL_Surface **surface, GlyphMetric **metric) {
```

Comment:
Would it make sense to just split this into two functions, like `font_group_get_glyph_surface` and `font_group_get_glyph_metrics`?
I guess, since getting the surface implies also getting the metrics, getting the surface should also return the metrics maybe.
Same thing for `font_load_glyph`.